jackpotlte/device.mk

```diff
@@ -0,0 +1,79 @@
+DEVICE_PACKAGE_OVERLAYS += $(LOCAL_PATH)/overlay
+
+TARGET_LOCAL_ARCH := arm64
+TARGET_SOC := exynos7885
+TARGET_TEE_KERNEL := mobicore
+
+# Inherit common device configuration
+$(call inherit-product, device/samsung/universal7885-common/universal7885-common.mk)
+
+$(call inherit-product, vendor/samsung/jackpotlte/jackpotlte-vendor.mk)
+
+$(call inherit-product, frameworks/native/build/phone-xhdpi-4096-dalvik-heap.mk)
+
+# device tree for boot image
+PRODUCT_HOST_PACKAGES += \
+    dtbhtoolExynos
+
+# USB
+PRODUCT_PACKAGES += \
+    android.hardware.usb-service.samsung
+
+TARGET_SCREEN_HEIGHT := 2220
+TARGET_SCREEN_WIDTH := 1080
+
+# Init
+PRODUCT_PACKAGES += \
+    init.exynos7885.rc.jackpotlte
+
+# Fingerprint
+PRODUCT_COPY_FILES += \
+    frameworks/native/data/etc/android.hardware.fingerprint.xml:$(TARGET_COPY_OUT_VENDOR)/etc/permissions/android.hardware.fingerprint.xml
+
+# Sensors
+PRODUCT_PACKAGES += \
```

Comment:
Alot of these sensor and camera packages dont need to be.
Need to test removing many of these on jackpot. For now it works, so that is cleanup.